data/words.txt

```diff
@@ -0,0 +1,4 @@
+cat
+dog
+rabbit
+wolf
```

Comment:
it is better to move this file to specs and move "data" folder to .gitignore or just delete it
and for all comands like load from file and save to file:
use commands format like
load -p "path/to/file.txt" [--zip]
save -d "path/to/destination" [--zip]